### PR TITLE
Fix image rotation on upload

### LINKED ISSUE #422: Image rotation doesn't work
Reported in detail here: https://publiclab.org/notes/aemilius89/12-12-2018/uploaded-pictures-always-stays-vertical

I think we could solve this pretty quickly -- just need to remember to do so!

### PR DESCRIPTION
**Pull request by gr455**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!
Fixes #422
Peek 2020-02-28 00-22

* [x] tests pass -- `rake test`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request are descriptively named
* [x] if possible, multiple commits squashed if they're smaller changes
* [x] reviewed/confirmed/tested by another contributor or maintainer

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

File: app/controllers/spectrums_controller.rb [hunk 1]
```diff
@@ -238,7 +238,7 @@ def create
               #@spectrum.tag("mobile", user.id)
             end
  
-            if params[:spectrum][:json].nil? && params[:vertical] == "on"
+            if params[:spectrum][:json]=="" && params[:vertical]
```
Comment (alaxalves) [inline app/controllers/spectrums_controller.rb]: Maybe adding some spaces here would be nice :) 
``` ruby
params[:spectrum][:json] == ""
```

Comment (gr455) [reply app/controllers/spectrums_controller.rb]: Done!

**Comment by gr455** [comment]
Squashed commits, ready for merge

**Comment by cesswairimu** [comment]
@gr455 can we please have a gif f this?

**Pull request merged**